src/components/policy/policy_external/src/policy_manager_impl.cc

```diff
@@ -550,6 +555,10 @@ void PolicyManagerImpl::OnPTUFinished(const PtProcessingResult ptu_result) {
 
   update_status_manager_.OnValidUpdateReceived();
 
+  if (HasApplicationForPTU()) {
```

Comment:
is this condition independent from `ptu_result`?

Reply:
@mked-luxoft No, no matter what the `ptu_result` will be from last update, we need set status to UPDATE_NEEDE for next ptu